src/pages/docs/creators/python/guides/index.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Golem Python Guides
+title: Golem Python Guides
+type: Introduction
+---
+
+Our Services API provides an abstraction over Golem low-level APIs, which is aimed at making the building of service-oriented applications straightforward for a developer. The abstraction is based on a logical concept of a Service, in other words, an entity that implements the logic of a service application, and which, from Requestor's perspective, follows a certain sequence of states:
```

Comment:
```suggestion
Our Services API provides an abstraction over Golem low-level APIs, which is aimed at making the building of service-oriented applications straightforward for a developer. The abstraction is based on a logical concept of a Service, in other words, an entity that implements the logic of a service application, and which, from Requestor's perspective, follows a certain sequence of states.
```

looks like there was a diagram here to which the colon was an introduction ;)
If we're skipping the diagram here, let's change it to a full stop ;)